bitnami/mediawiki/1/debian-12/rootfs/opt/bitnami/scripts/mediawiki/setup.sh

```diff
@@ -32,7 +32,10 @@ fi
 mediawiki_validate
 
 # Update web server configuration with runtime environment (needs to happen before the initialization)
-web_server_update_app_configuration "mediawiki"
```

Comment:
if we remove this here, i don't think `web_server_update_app_configuration` is used anywhere

Reply:
Both `ensure_web_server_app_configuration_exists` and `web_server_update_app_configuration` serve different purposes.

`ensure_web_server_app_configuration_exists` initializes the Apache configuration at image build time, while `web_server_update_app_configuration` updates it at runtime to configure the server-specific host and port configured via env vars.

It is possible to replace `ensure_web_server_app_configuration_exists` from build-time to run-time, but you would need one of the following:
- Keep `web_server_update_app_configuration` after `ensure_web_server_app_configuration_exists`.
- Add args `--http-port ` and `--https-port` to `ensure_web_server_app_configuration_exists`.

I would personally recommend option 1.